Quickstart examples/fargate/README.md

```diff
@@ -0,0 +1,6 @@
+quickstart-mongodb-atlas-mean-stack-aws-fargate-integration—Quick Start
+For architectural details, step-by-step instructions, and customization options, see the deployment guide.
+
+To post feedback, submit feature ideas, or report bugs, use the Issues section of this GitHub repo.
+
+To submit code for this Quick Start, see the AWS Quick Start Contributor's Kit.
```

Comment:
- Can we add the other readme files in the client and server?
- Can we add the link to the step-by-step instruction?

Reply:
@poornatejakonathampi @Adelmar92 ^^

Reply:
Added build scripts for both client and server applications. Updated readme file.